l10n/de.js

```diff
@@ -31,6 +31,7 @@ OC.L10N.register(
     "Address books" : "Adressbücher",
     "Download" : "Herunterladen",
     "Send email" : "E-Mail senden",
+    "Send email as bcc" : "E-Mail als bcc senden",
```

Comment:
translations are handled via Transifex. Please undo this change. the new string will be sent to and pulled from transifex with a daily cron job